src/test/run-make-fulldeps/coverage-reports/expected_show_coverage.uses_crate.txt

```diff
@@ -19,12 +19,12 @@
    18|      2|    println!("used_only_from_bin_crate_generic_function with {:?}", arg);
    19|      2|}
   ------------------
-  | used_crate::used_only_from_bin_crate_generic_function::<&alloc::vec::Vec<i32>>:
```

Comment:
@wesleywiser - Ignore the change here. When @Swatinem pushed his recent changes, built from his Windows dev environment, the order of these variants got swapped in the output. (Windows known issue, but tests pass either way.)

When I re-built the blessed results, it swapped them back again.